### PR TITLE
[JENKINS-48353] - Use Apache HttpComponents Client 4.x API Plugin as a client source

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Just replaces the local libraries by a plugin dependency

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
@reviewbybees @i386

**Review by dwnusbaum — APPROVED** [review]
🐝

File: pom.xml [hunk 1]
```diff
@@ -250,36 +253,9 @@
...
-        </exclusion>
-      </exclusions>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.httpcomponents</groupId>
-      <artifactId>fluent-hc</artifactId>
-      <version>${httpclient.version}</version>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>apache-httpcomponents-client-4-api</artifactId>
+      <version>4.5.3-2.0</version>
```
Comment (rantoniuk) [inline pom.xml:258]: @oleg-nenashev what's that weird versioning scheme? :-)

Comment (oleg-nenashev) [reply pom.xml:258]: * 4.5.3 - version of Apache HttpComponents Client 
* 2.0 - Number of the release

**Review by rantoniuk — APPROVED** [review]

**Comment by i386** [comment]
Nice! Thanks for this change

**Pull request merged**